src/styles/product-card.css

```
@@ -0,0 +1,34 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  border-radius: 5px;
+  padding-bottom: 16px;
+}
+
+.product-card__image {
+  margin: 32px auto 0;
+}
+
+.product-card__title {
+  margin: 40px var(--margin-side) 0;
+}
+
+.product-card__product-code {
+  margin-left: var(--margin-side);
+  margin-top: 4px;
+}
+
+.product-card__price {
+  margin: 24px var(--margin-side) 0;
+}
```

just us margin-bottom or margin-top (depends on your code styling)
```suggestion
.product-card__title {
  margin-bottom: 4px;
}

.product-card__product-code {
  margin-bottom: 24px;
}

.product-card__price {
  margin-bottom: 16px;
}
```